site/docs/how-to/aws.mdx

```diff
@@ -49,3 +49,36 @@ SELECT
 FROM
   iasql_install ('aws_iam', 'aws_cloudwatch', 'aws_ecr', 'aws_ecs_fargate', 'aws_elb', 'aws_security_group', 'aws_vpc');
 ```
+
+## Restricting AWS credentials to specific regions
+
+Sometimes, you may want or need to restrict your credentials to a specific region. To do this, you will need to create a custom policy that restricts access to AWS resources in that region. The following [guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html) will walk you through creating a new policy. Your policy will need to look something like this:
```

Comment:
I'm assuming we don't have the ability to do this in IaSQL itself yet, right?

If we do, we *may* want to also include that, though noting that you probably don't want to edit your own IAM identity this way, because we may lose the ability to perform the operations in the middle of performing said operations, and that it should be done from a different account.

Reply:
hmm, we do not have policies management yet, but we do have a field in `user` and `role` entities that let you update the attached policies.

Reply:
We can create that policy as a public policy in one of our account and use the ARN to attach that through IaSQL. That's a bit hacky and if we want to use a non-hacky way we can add the support for IAM policies.

Reply:
I think it can make sense to have both views, one for "you already have a restricted IAM account and need to keep IaSQL from trying to access regions it's not allowed to" which really only needs the last part, and the other view of "I am trying to set up a restricted IAM user to then use with IaSQL" which could have both "do it in the console" and "use my powerful IAM user to create the less powerful IAM user" explanations.

I guess what *this* particular article is aiming at is the question, but I assumed it was for people who already had restricted IAM accounts.

Reply:
The first section of this guide already covers how to create the IAM credentials or how to associate them and including this modification around that seems sensible. I don't think that we need to lay out again how to provision an IAM role again or whether they should create a new one. We can simply say the credentials provided need a policy that looks like this:

```suggestion
Sometimes, you may want or need to restrict access to a specific region. To do this, you will need to make sure that the credentials for the IAM role given to IaSQL have a policy that restricts access only to AWS resources in that region. This can be done by creating a custom policy and attaching it to the relevant IAM role. The following [guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html) will walk you through creating a new policy. Your policy will need to look something like this:
```

Is it possible to attach the policy inline?

Reply:
yes, you can